sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriterOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.ClientModel
+{
+    /// <summary>
+    /// Provides the client options for reading and writing models.
+    /// </summary>
+    public class ModelReaderWriterOptions
+    {
+        private static ModelReaderWriterOptions? _jsonOptions;
```

Comment:
nit: The convention for the repo is using `s_` to mark static member variables.   As a disclaimer, I absolutely hate it but felt it was worth mentioning.

_(that said, I have no objection if we "won't fix" )_